core/templates/dev/head/pages/exploration_editor/ExplorationSaveAndPublishButtons.js

```diff
@@ -13,98 +14,520 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controller for the exploration save & publish buttons.
+<<<<<<< HEAD
```

Comment:
There are some serious issues here, in addition to what looks like a merge conflict on this line. I think it's because this PR took a long time to get to a "first draft" stage, and @Keksike submitted a PR in the meantime where he refactored the save/publish buttons.

Could we please get rid of the save/publish button stuff from this PR, in order to keep the complexity manageable?